src/render/filters.rs

```diff
@@ -299,6 +360,71 @@ mod tests {
         })
     }
 
+    #[test]
+    fn test_render_filter_slugify() {
```

Comment:
I think we should probably split this particular test up into multiple tests. If anything breaks for whatever reason, I'd rather see which of these scenarios fail rather than just one.